bin/daps-xmlformat

```diff
@@ -25,9 +25,16 @@ fi
 #echo $cmd
 #eval $cmd
 
-res=$(which xmlformat.pl 2>/dev/null)
+# lets decide on the binary filename
+dist=`lsb_release -ds | awk '{print $1}'`
```

Comment:
I'm not sure if this is the "right" test to check for Debian. Maybe it's enough to check for some Debian specific files? In openSUSE we have `/etc/SuSE-release` and `/etc/SuSE-brand`, maybe there is something similar?